regression/goto-analyzer/monotonic_change_values/main.c

```diff
@@ -0,0 +1,22 @@
+int nondet(void);
+
+int main()
+{
+  int x = 42;
+  x++;
+  x = x + 2;
+
+  int y = 42;
+  y--;
+  y = y - 2;
+
+  int z = (y + 1729) * (x - 6);
+  z++;
+  z--;
+
+  int u = nondet();
```

Comment:
Can we write:
```suggestion
  int u;
```
and remove the `nondet` declaration? Variables that are declared (but not defined) are nondet by default